esx_service/vmdk_ops_test.py

```diff
@@ -722,15 +722,21 @@ def create_default_tenant_and_privileges(self):
 
         if not tenant_uuid:
             logging.debug("create_default_tenant_and_privileges: create DEFAULT tenant")
-            error_info, tenant = auth_api._tenant_create(
+            error_info, auth_mgr = auth_api.get_auth_mgr_object()
+            if error_info:
+                err = error_code.TENANT_CREATE_FAILED.format(auth.DEFAULT_TENANT, error_info.msg)
```

Comment:
it is better to say "Error occurred while getting auth manager object" instead of TENANT_CREATE_FAILED at this point that help in failure triaging.